src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -4,7 +4,7 @@
   <!-- Official Version -->
   <PropertyGroup>
     <MajorVersion>2</MajorVersion>
-    <MinorVersion>3</MinorVersion>
+    <MinorVersion>4</MinorVersion>
```

Comment:
We should only update the base package version and release that first. Once Microsoft.FeatureManagement v2.4.0 is on NuGet, we can bump up the version of AspNetCore project and also update the base package reference at the same time.

Reply:
Why is that?

Reply:
The AspNetCore project does not reference the base project by package version.
https://github.com/microsoft/FeatureManagement-Dotnet/blob/423501a59d8d086bbe60baa89dedcded1133213f/src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj#L33

Reply:
Oh I see, in that case we don't need separate version upgrades because AspNetCore project has a reference to the base project directly. 
For DotNetProvider, we need two different version upgrades because AspNetCore project references the latest base package from NuGet.